samples/boards/96b_carbon/multi_thread_blinky/README.rst

```diff
@@ -0,0 +1,37 @@
+.. _96b_carbon_multi_thread_blinky:
+
+96Boards Carbon Basic Multi Thread Example
+##########################################
+
+Overview
+********
+
+This example demonstrates spawning of multiple threads using K_THREAD_DEFINE.
+
+The example works by spawing three threads each to controll a separate LED 
+on the 96Boards Carbon Board. Each of these LEDs (USR1, USR2 and BT) have 
+their individual loop control and timing logic defind inside separate functions.
+
+blink1() controls the USR1 LED that has a 100ms sleep cycle
```

Comment:
Let's make this a bullet list by adding a ``-`` before each new item, and from the code it looks like blink3 never makes it to 600ms so:

     - blink1() controls the USR1 LED with a 100ms on/off blink rate
     - blink2() controls the USR2 LED with a 1000ms (1 sec) on/off blink rate
     - blink3() controls the BT LED with a 100ms on/off blink rate that
       increases by 100ms up to 500ms, then resets back to 100ms